Electron/src/pages/UserProfile/UserProfile.tsx

```diff
@@ -266,32 +273,40 @@ export default function UserProfile({
 
       {userType === UserType.USER && (
         <div className="p-4">
-          <h2
-            style={{
-              color: 'var(--pure-white)',
-              fontWeight: '700',
-              fontSize: '1.5rem',
-              marginTop: '1rem',
-              marginBottom: '1.5rem',
-            }}
-          >
-            Historial de reproducción
-          </h2>
-          <div className="d-flex flex-row flex-wrap " style={{ gap: '14px' }}>
-            {playbackHistory &&
-              playbackHistory.map((songItem, index) => {
-                return (
-                  <SongCard
-                    // eslint-disable-next-line react/no-array-index-key
-                    key={`${songItem.name}-${index}`}
-                    name={songItem.name}
-                    photo={songItem.photo}
-                    artist={songItem.artist}
-                    changeSongName={changeSongName}
-                    refreshSidebarData={refreshSidebarData}
-                  />
-                );
-              })}
+          <div className="d-flex">
```

Comment:
We want to do the same as before but adding `Mostrar todos`. This means, showing 5 items as preview and then by clicking `Mostrar todos` the allitems component shows all the 50 songs at the same time

Reply:
Please, check latest commit, it should be implemented there as you proposed